recipes/libcdms/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "libcdms" %}
+{% set version = "2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/UV-CDAT/libcdms/archive/master.tar.gz
+  sha256: 5456bfa2ac3b5ff7d0d05fce7ac2da3d2cdf08017a4f4019c99ada91f7e48649
+
+build:
+  number: 0
+  skip: True  # [win] 
+
+requirements:
+  build:
+    - hdf5
+    - libnetcdf
```

Comment:
Please you pin the dependencies according to https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies